src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/collectors/ClusterApplierServiceStatsCollector.java

```diff
@@ -46,7 +46,7 @@
     private static final Logger LOG = LogManager.getLogger(ClusterApplierServiceStatsCollector.class);
     private static final String GET_CLUSTER_APPLIER_SERVICE_STATS_METHOD_NAME = "getStats";
     private static final ObjectMapper mapper;
-    private static volatile ClusterApplierServiceStats prevClusterApplierServiceStats = new ClusterApplierServiceStats();
+    private volatile ClusterApplierServiceStats prevClusterApplierServiceStats = new ClusterApplierServiceStats();
```

Comment:
why remove `static` here?

Reply:
findbugs was failing saying write toa  static field. Also it would not break my use case , since we are instantiating the collector only once and collect metrics method is called at pre-configured intervals